src/components/accessories/patientPicker/PatientPicker.tsx

```diff
@@ -61,7 +51,16 @@ const PatientPicker: FC<IProps> = ({
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const inputRef = useRef<any>(null);
-
+  const [currentPage, setCurrentPage] = useState(1);
+  const [patientsPerPage] = useState(5);
```

Comment:
Consider to promote those values as constants in a dedicated file.

Reply:
OK